context/app/static/js/components/Search/SelectedFilter/style.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import CancelIcon from '@material-ui/icons/CancelRounded';
+
+const StyledCancelIcon = styled(CancelIcon)`
+  height: 1.2rem;
+  margin: 0 8px;
+  color: ${(props) => props.theme.palette.primary.main};
+  cursor: pointer;
+`;
+
+const SelectedFilterDiv = styled.div`
+  background-color: white;
+  border: 1px solid ${(props) => props.theme.palette.hoverShadow.main};
+  border-radius: 8px;
+  margin: 16px 16px 0 0;
+  display: flex;
+  padding: 10px 0 10px 10px;
+  font-size: 14px;
```

Comment:
Use `Typography` for the text inside the div to keep things consistent and use theme values by default?